### ISSUE TITLE
Cant set Column Width's in one try

### ISSUE THREAD

**Issue by JeremyBYU** [jrief/djangocms-cascade#24]
Okay so I figured out how to change the  column width on the cms. However I noticed a small little bug (at least I think it is).
For example if you have a Default Width  (xs-col) and it is set at 4 already, and you want to change it to 12 you can.  But if you want to change the width for the "Width for Devcies > 768 px" to 6 it will not be possible, it will only let you go to 4.  
However if you save the form.  Then open it up again it will now let you go to 6.  

Basically the default width limits the choices of the other dropdowns.  However if you change that defualt width it will not update the available options in the other dropdowns.  You will have to save the form then reopen again. 
![cms_cascade_error](https://cloud.githubusercontent.com/assets/6628085/3484943/d1cc80b4-03bb-11e4-923b-e2e34f709120.png)

**Comment by rfleschenberg** [comment]
I think this issue has been fixed. Closing.
